libfreerdp/core/server.c

```diff
@@ -1033,6 +1038,7 @@ HANDLE WINAPI FreeRDP_WTSVirtualChannelOpen(HANDLE hServer, DWORD SessionId, LPS
 
 	if (!channel)
 	{
+		wObject queueCallbacks;
```

Comment:
Missing initialization `wObject queueCallbacks = { 0 };`

Reply:
fixed it, thank you :))